src/components/DataCollector/index.js

```diff
@@ -33,7 +33,8 @@ const createDataCollector = ({ eventManager, logger }) => {
           viewStart = false,
           documentUnloading = false,
           type,
-          mergeId
+          mergeId,
+          scopes = []
```

Comment:
I know `scopes` is what we named those things in the decisions schema, but do you think we need a more descriptive name for the `event` command?

cc: @Aaronius  @artur-ciocanu  @jonsnyder

Reply:
That's a good point. I think it should be more specific. Something like `personalizationScopes` or whatever @artur-ciocanu and @ninaceban feel like is most appropriate.

Reply:
`scopes` is an array of locations, for example: ["banner", "footer", "header"], I don't have something in mind right now for the `event` command, something that would be a cross solution name and better than `event`

Reply:
Let's discuss this in today's standup.